fcl/gen/numi/simulation_genie_icarus_numi_rock.fcl

```diff
@@ -0,0 +1,8 @@
+#include "simulation_genie_icarus_numi.fcl"
+
+# 
+# overrides
+#
+physics.producers.generator: @local::icarus_genie_NuMI_rock
+physics.producers.generator.TopVolume:  "volWorld"
+physics.producers.generator.FiducialCut: "rockbox:(-571.55,-352.06667,-1187.037)(+571.55,+619.03333,+1566.203),1,550,0.00425,1.05,1"
```

Comment:
Not sure exactly where to put this comment, but putting it here for lack of a better place.

I am thinking maybe it would be good to put some doc-db number in a comment somewhere pointing out where the configuration came from or was tested. I talked to Minerba who said a combination of a doc-db from a recent test plus her former intern's work could be put together that would go into that comment.

Reply:
I'm happy to put in a reference to a presentation if there is one -- do you have a docdb #?

Reply:
Not at the moment. Let me ask when we should expect one to see if this should go in this PR or just be tacked on later.

Reply:
I'd like the PR to be merged tomorrow for this week's release. So either I need it now or we can add it to a future PR.

Reply:
That's basically what I said above. In any case when we took the conversation to Slack, it resulted in the change you just committed, so I'm happy that I asked.

Will there be a release tomorrow given there was a release cut yesterday?